CONTRIBUTING.md

```diff
@@ -1,52 +1,72 @@
-
-Hello there! So glad you want to help out with the LabVIEW Icon Editor 🤗 You’re the best. Here’s a guide for how you can contribute to the project.
-
 # The Open Source LabVIEW Icon Editor
 
-Just recently, the LabVIEW Icon Editor has been made open source to encourage collaboration between NI and the LabVIEW Community. Now you can find the source code of the LabVIEW Icon Editor in this repository. We are in a state of transition. This is the current structure:
-
-* LabVIEW Icon Editor `labview-icon-editor` version {latest_version} is on branch {branch_name}
-* The latest version of the LabVIEW Icon Editor (not published yet) can be found {latest_version_link}
-
-# Our Contributing Philosophy
+Just recently, the LabVIEW Icon Editor has been made open source to encourage collaboration between NI and the LabVIEW Community. Every new build of LabVIEW will grab the icon editor that has been pushed into the main branch from this repo.
 
 The direction of the LabVIEW Icon Editor and which new features are added, and which are left out, is decided by the core team, which includes NI staff and LabVIEW community volunteers (sometimes referred to as a “cathedral” style of development).
 
 That being said, the Icon Editor is great because of the LabVIEW users who use it, share their code with the community, and discuss ways to make LabVIEW even better. Some of the most important and undervalued work in open source is from non-code contributions, and that is where we can use the most help from you.
 
+# Our Contributing Philosophy
+
+This repo is managed via [git](https://git-scm.com), with the canonical upstream repository hosted on [GitHub](https://github.com/ni/labview-icon-editor) and it follows a pull-request model for development. If you wish to contribute, you will need to create a GitHub account, fork this project, push a branch with your changes to your project, and then submit a pull request.
+
 # How can I help?
 
 The things we need the most help for the library and its community are:
 
 **🐛 Bug reports:** We simply can’t catch them all. Check [existing issues](https://github.com/ni/labview-icon-editor/issues/new/choose) and discussion first, then [create a new issue](https://github.com/ni/labview-icon-editor/issues/new/choose) to tell us what’s up.
 
-**💬 Answering questions** in our {discussion_form} or {community_discord}.
+**💬 Answering questions and provide feedback**: New features begin by creating a discussion on our board, you can provide feedback to the new features that are planned to be added to the icon editor.
 
-🎬 **Create tutorials**. Send them to us and we’ll happily share them!
+🎬 **Work on "Good first issue" items**: You can right now assign yourself action items to finish or test a feature. Check out the issues section and look for any issue labeled as "Good first problem" if you want to passively contribute to the repo, or if you are feeling adventurous you can always assign problems not labeled as "Good first issue".
```

Comment:
Make "good first issues" text a link to the issues list, filtered to only show "good for issues" @svelderrainruiz

Reply:
Like this [Good First Issues](https://github.com/ni/labview-icon-editor/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```[Good First Issues](https://github.com/ni/labview-icon-editor/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)```